matplotcheck/base.py

```diff
@@ -360,12 +389,13 @@ def assert_equal_xlims_ylims(self, m="xlims and ylims are not equal"):
 
         Parameters
         ---------
-        m: string
+        m : string
```

Comment:
similarly let's update this throughout!!

Reply:
Will address this in a future PR